pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -18,6 +18,7 @@
 , passthru ? {}
 , extraPreBwrapCmds ? ""
 , extraBwrapArgs ? []
+, multiArch ? false # Whether to include 32bit packages
```

Comment:
Is defaulting to false intended?

Reply:
Yes, it defaults to false in the core build function `buildFHSEnv` too.